test/service/internal_key_event_handlers/format_style_handler_test.dart

```diff
@@ -246,7 +246,27 @@ Future<void> _testLinkMenuInSingleTextSelection(WidgetTester tester) async {
     ..insertTextNode(text);
   await editor.startTesting();
 
-  final selection =
+// selection is collapsed
+  final emptySelection =
+      Selection.single(path: [0], startOffset: 7);
+  await editor.updateSelection(emptySelection);
+
+  // Link dialog should not be visible when selection is null or collapsed
+  if(Platform.isWindows || Platform.isLinux){
+    await editor.pressLogicKey(
+      key: LogicalKeyboardKey.keyK,
+      isControlPressed: true,
+    );
+  } else {
+    await editor.pressLogicKey(
+      key: LogicalKeyboardKey.keyK,
+      isMetaPressed: true,
+    );
+  }
```

Comment:
And here, missing space between `if` and `(`.

Also there is an easier way to do this:
```suggestion
  await editor.pressLogicKey(
    key: LogicalKeyboardKey.keyK,
    isControlPressed: !Platform.isMacOS
    isMetaPressed: Platform.isMacOS,
  );
```